WindowsServerDocs/get-started/editions-comparison-windows-server-2022.md

```diff
@@ -44,6 +44,8 @@ Use this article to compare Standard, Datacenter, and Datacenter: Azure Edition
 |Inherited activation|As guest if hosted on Datacenter| **Can be a host or a guest** | **Can be a host or a guest** |
 |Work Folders|Yes|Yes|Yes|
 
+<sup>1</sup> - Support may be limited to on-premises installations with compatible networking hardware. Windows Server 2022 Datacenter: Azure Edition has RDMA support but can only be run in Azure and Azure Stack HCI. Please refer to [Azure documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/hpc/enable-infiniband) for current RDMA support requirements. RDMA can be used in Azure Stack HCI when following network [switch](https://docs.microsoft.com/en-us/azure-stack/hci/concepts/physical-network-requirements) and [host](https://docs.microsoft.com/en-us/azure-stack/hci/concepts/host-network-requirements) requirements and [best practices](https://docs.microsoft.com/en-us/azure-stack/hci/concepts/network-atc-overview). 
```

Comment:
Hyper-V and Azure Stack HCI folks don't support guest RDMA, might want to make that clearer (i.e. even if AE supports RDA, the hosts won't let them use it). @dcuomo can add more clarifying about this and what Azure will support.

Reply:
Thanks @nedpyle, I'll reach out to Dan. 

@JamesKehr - I've pulled this commit to our private repo to continue the edits. Please let me know if you have any questions or need to make additional edits.

Thanks,
Robin